### ISSUE TITLE
ME:Andromeda Issue

### ISSUE THREAD

**Issue by Reaper507** [GalaxyEham/FrostyToolSuite#161]
1. There is no error. Mod Manager cann't start ME:Andromeda.
2. Mod Manager.
3. Lastest version 1.0.5.1.
4. Mass Effect Andromeda.
5. Windows 7 professional 64 bit.
6. nVidia 1050 GTX.
7. Launch Mod Manager with administrative privileges, select Mass Effect Andromeda executable. Empty mod list. Or it can contains some mods. It doesn't matter. Press "Launch" button. Mod Manager starts Andromeda process, but Andromeda process hangs. It not depends on a mod list.
8. ---
9. ---
10. ---

You wrote about permissions in closed ticket, but I do not know what else to do besides running the MM with administrator rights. Old version 1.0.3.a4 works fine. It runs ME:Andromeda with mods. ME:Andromeda starts fine standalone. It is not a problem with GPU. It is not a problem with mod list. 

I've read, you wrote it works on your computer. I don't know what difference between our systems. I use C:\Games\ for installed games. "Users" have rights for this directory. Your Mod Manager (all versions) creates files in ME:A\ModData\ and lastest version copies CryptBase.dll into the game root directory. It seems like all ok with permissions.

**Comment by GalaxyEham** [comment]
1. Make sure to check Run as Administrator for both Frosty and MEA's exe. Then delete the ModData folder in your MEA game files.
 2. Try moving your Frosty folder to a different drive.
 3. Take ownership of both your MEA game directory and the Frosty Mod Manager folder.
 4. If on Windows 10, add an exception to Windows Defender or any form of antivirus for Frosty.
 5. Restart Origin and your PC.

Keep in mind that these fixes are not guaranteed to work. Every setup is different, so you might just have to mess around.

NOTE: Always make sure to delete the ModData folder in your nfs Game Files every time you try something different.

and before you say you have tried them all and it doesnt work, I will repoint you to the text above the NOTE. 

MEA works fine with the Mod Manager, on all operating systems supported by Frosty, as has been shown by the numerous people who can use it successfully, for the few of you who cannot, it is entirely your PC configuration. The steps are suggestions, and are not a guaranteed fix, as the note states, every configuration is different, and so therefore the potential root cause of your issue is potentially going to be different.

Just because an old version worked does not guarantee that new versions wont have additional issues, as new versions have more functionality, and can conflict with your configuration in various other ways.

It is also not recommended to run these old versions as new versions are created for a reason, namely to fix issues with the old versions, if you do wish to continue running the old versions, I will not provide any support for them what so ever.

**Comment by Reaper507** [comment]
I tried the recommended steps, no success. And yes, I didn't forget to delete the folder during the attempts. 

Ok, taken: it's all my fault. It is something with my configuration, may be aliens cursed it or solar flares. No sense to stole your time anymore, cuz as software developer I understand: if old version works and new version fails it means only one: a problem on user side, not developer. It exactly what I say them.

At least exists the old version, there are no other possibilities to play ME:A with mods for "my configuration". Best regards anyway.

**Comment by GalaxyEham** [comment]
and being a smart arse aint helpful, when an old version works, and a new version doesnt, there are a myriad of potential reasons, as a supposed Software Developer yourself, you should already know that. It could be incompatabilities with your operating system, computer configuration. It could be missing pre-reqs, because the new version might have a different set of pre-reqs, it could be permissions as has already been stated. 

Basically the new version has already been tested across a large sample of configuration types, and it works fine, so again, yes I say to you, the issue is your configuration in particular.

As for using the old version, eventually that will no longer be feasible, infact it is not now, if and when people create new mods with 1.0.5.1, they are unusable in anything before 1.0.5, so its best to fix your issue.

**Comment by Ghecko166** [comment]
GalaxyEham, I strongly suggest you read this forum: there are MANY users who have issues with FFM and MEA.

https://www.nexusmods.com/masseffectandromeda/mods/224/?tab=posts

Saying "it works for me" and attacking those who tell you otherwise is not the best way to solicit feedback.

**Comment by GalaxyEham** [comment]
And others have also had the issue and 100% have resolved it via the same steps I provided above, or via finding whatever was incompatible or blocking execution. So before you continuously dismiss it and decide that I have some kind of vendetta against every single person who has an issue, maybe you should actually try to figure out what the incompatibility is. As I cannot tell you, as only you know your configuration. I can only point out that "it works for me" and it works for a hell of a lot of other people.

In no way was I attacking you, I was purely pointing out that trying to be clever with your insults to me, will just result in me not wanting to help

**Comment by BLaDZer** [comment]
**BUG**
APP: FrostyModManager v1.0.5 and higher
GAME: ME:Andromeda 1.10
SYSTEM: Windows 7 x64
GRAPHICS: nVidia 1060 GTX
PROBLEM: game runs in background with ~8 MB memory and stays there even if no mods are used

Hello @GalaxyEham I hope we can still continue solving this issue?
Same problem with game and after spending a few hours I found this issue and tried an older FFM versions 1.0.3 and 1.0.4 (ofc after deleting moddata folder) and them both working!

Maybe it's Win7 issue? Tell us how we can help you to fix this? App dump or maybe you have a debug version with log output... we can test version with reverted some functions you have added/changed in this release ... or something else?

I hope for your understanding and help.

**Comment by t14462** [comment]
1.0.5.3 Launching fine DAI, but MEA running as background process, and nothing happen.

Tried all methods already, and old versions - good with DAI, nothing with MEA

**Comment by nguyencongthienvu** [comment]
> **BUG**
> APP: FrostyModManager v1.0.4.5 and higher
> GAME: ME:Andromeda 1.10
> SYSTEM: Windows 8.1 x64
> GRAPHICS: nVidia 1060 GTX
> PROBLEM: game runs in background with ~8 MB memory and stays there even if no mods are used
> 
> Hello @GalaxyEham I hope we can still continue solving this issue?
> Same problem with game and after spending a few hours I found this issue and tried an older FFM versions 1.0.3.1 and 1.0.4.4 (ofc after deleting moddata folder) and them both working!
> 
> Maybe this bug only happens on Windows below Win10? Tell us how we can help you to fix this? App dump or maybe you have a debug version with log output... we can test version with reverted some functions you have added/changed in this release ... or something else?
> 
> I think it could be because of:
> 
> > This version introduces a new method for allowing modded games to be ran. Just to be on the safe side, make sure to install Visual C++ Runtimes 2017. Also if you are on Windows 8 or below and have any issues with launching the games, please report in #troubleshooting on the Frosty server.
> 
> or
> 
> > Fixed an issue with Windows 10 update 1809 unable to launch modded games
> 
> I hope for your understanding and help.

This right, v1.45 and later it not lauch game special ME and SWBF2 (without mod add) on win 7,8. I think because the code fix win 10 or method C++ 2017 make function for win 7, 8 stuck. Please fix that, thank you.

**Comment by SiggeSigvardsson** [comment]
> 
> 
> **BUG**
> APP: FrostyModManager v1.0.4.5 and higher
> GAME: ME:Andromeda 1.10
> SYSTEM: Windows 8.1 x64
> GRAPHICS: nVidia 1060 GTX
> PROBLEM: game runs in background with ~8 MB memory and stays there even if no mods are used
> 
> Hello @GalaxyEham I hope we can still continue solving this issue?
> Same problem with game and after spending a few hours I found this issue and tried an older FFM versions 1.0.3.1 and 1.0.4.4 (ofc after deleting moddata folder) and them both working!
> 
> Maybe this bug only happens on Windows below Win10? Tell us how we can help you to fix this? App dump or maybe you have a debug version with log output... we can test version with reverted some functions you have added/changed in this release ... or something else?
> 
> I think it could be because of:
> 
> > This version introduces a new method for allowing modded games to be ran. Just to be on the safe side, make sure to install Visual C++ Runtimes 2017. Also if you are on Windows 8 or below and have any issues with launching the games, please report in #troubleshooting on the Frosty server.
> 
> or
> 
> > Fixed an issue with Windows 10 update 1809 unable to launch modded games
> 
> I hope for your understanding and help.


Game: Andromeda 1.10 Delux
OS: Win 10 1903 and 1909
GPU: 2060 super, drivers 436/441


Having the same issues, but with the diffrence they all start it in the background. 
With the exception, starting the game with no mods loaded/installed works in 1.0.3alpa. But as soon as i add a mod it will not start. 

Tried different bcrypt files, no bcrypt. Every version since 1.0.3alpa untill 1.5.7. 
Owner of the files and is running as admin. And MEs exe is also set as admin.  (Tried both)

Reinstalled the game, on diffrent drives C: G:. 

Game works with no problem running it without frosty.

**Comment by KonstantinRitt** [comment]
Game: Andromeda 1.10 Delux
OS: Win 7 Ultimate SP1

Same issue with since FrostyModManager 1.0.4.5 thru 1.0.5.9. Runs just fine with FrostyModManager 1.0.4.2.
